client/components/forms/form-text-input/index.jsx

```diff
@@ -19,25 +19,31 @@ module.exports = React.createClass( {
 		};
 	},
 
-	render: function() {
-		var otherProps = omit( this.props, [ 'className', 'type' ] ),
-			classes = classNames( {
-				'form-text-input': true,
-				'is-error': this.props.isError,
-				'is-valid': this.props.isValid
-			} );
+	focus() {
+		ReactDom.findDOMNode( this.refs.textField ).focus();
```

Comment:
Part of the benefit of moving `focus` to `<FormTextInput />` is that now you have access to the "raw" DOM element, so you no longer need `findDOMNode`. Here, `this.refs.textField` is the a reference to the DOM node itself, so this can simply be `this.refs.textField.focus()` and you can drop the `react-dom` dependency.